2022-10-16 15:45:42,167 INFO    MainThread:8472 [wandb_setup.py:_flush():68] Configure stats pid to 8472
2022-10-16 15:45:42,167 INFO    MainThread:8472 [wandb_setup.py:_flush():68] Loading settings from /Users/zhanghangsheng/.config/wandb/settings
2022-10-16 15:45:42,167 INFO    MainThread:8472 [wandb_setup.py:_flush():68] Loading settings from /Users/zhanghangsheng/Documents/my_code/Cyber-Security-ML-Toolbox/wandb/settings
2022-10-16 15:45:42,168 INFO    MainThread:8472 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-10-16 15:45:42,168 INFO    MainThread:8472 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'tests/sequence/test_classifer.py', 'program': '/Users/zhanghangsheng/Documents/my_code/Cyber-Security-ML-Toolbox/tests/sequence/test_classifer.py'}
2022-10-16 15:45:42,168 INFO    MainThread:8472 [wandb_init.py:_log_setup():476] Logging user logs to /Users/zhanghangsheng/Documents/my_code/Cyber-Security-ML-Toolbox/wandb/run-20221016_154542-3q09o07i/logs/debug.log
2022-10-16 15:45:42,168 INFO    MainThread:8472 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/zhanghangsheng/Documents/my_code/Cyber-Security-ML-Toolbox/wandb/run-20221016_154542-3q09o07i/logs/debug-internal.log
2022-10-16 15:45:42,168 INFO    MainThread:8472 [wandb_init.py:init():516] calling init triggers
2022-10-16 15:45:42,168 INFO    MainThread:8472 [wandb_init.py:init():520] wandb.init called with sweep_config: {}
config: {}
2022-10-16 15:45:42,168 INFO    MainThread:8472 [wandb_init.py:init():569] starting backend
2022-10-16 15:45:42,168 INFO    MainThread:8472 [wandb_init.py:init():573] setting up manager
2022-10-16 15:45:42,174 INFO    MainThread:8472 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-10-16 15:45:42,178 INFO    MainThread:8472 [wandb_init.py:init():580] backend started and connected
2022-10-16 15:45:42,181 INFO    MainThread:8472 [wandb_init.py:init():658] updated telemetry
2022-10-16 15:45:42,183 INFO    MainThread:8472 [wandb_init.py:init():693] communicating run to backend with 30 second timeout
2022-10-16 15:45:43,032 INFO    MainThread:8472 [wandb_run.py:_on_init():1974] communicating current version
2022-10-16 15:45:43,457 INFO    MainThread:8472 [wandb_run.py:_on_init():1978] got version response upgrade_message: "wandb version 0.13.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-10-16 15:45:43,457 INFO    MainThread:8472 [wandb_init.py:init():727] starting run threads in backend
2022-10-16 15:45:45,321 WARNING MainThread:8472 [wandb_init.py:init():1097] interrupted
Traceback (most recent call last):
  File "/opt/anaconda3/envs/py36/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 1076, in init
    run = wi.init()
  File "/opt/anaconda3/envs/py36/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 738, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/opt/anaconda3/envs/py36/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 235, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/opt/anaconda3/envs/py36/lib/python3.6/site-packages/wandb/sdk/interface/interface_shared.py", line 444, in _communicate_run_start
    result = self._communicate(rec)
  File "/opt/anaconda3/envs/py36/lib/python3.6/site-packages/wandb/sdk/interface/interface_shared.py", line 230, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/opt/anaconda3/envs/py36/lib/python3.6/site-packages/wandb/sdk/interface/router.py", line 36, in get
    is_set = self._object_ready.wait(timeout)
  File "/opt/anaconda3/envs/py36/lib/python3.6/threading.py", line 551, in wait
    signaled = self._cond.wait(timeout)
  File "/opt/anaconda3/envs/py36/lib/python3.6/threading.py", line 299, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2022-10-16 15:46:02,132 WARNING MsgRouterThr:8472 [router.py:message_loop():77] message_loop has been closed
